demo.py

```diff
@@ -0,0 +1,79 @@
+from github import Github
```

Comment:
This file could possibly be removed since the demo is already over. But a lot of these can go into the test cases if needed